tools/lint-all

```diff
@@ -402,6 +402,8 @@ def build_custom_checkers(by_lang):
     help_markdown_rules = markdown_rules + [
         {'pattern': '[a-z][.][A-Z]',
          'description': "Likely missing space after end of sentence"},
+        {'pattern': '[rR]ealm',
+         'description': "Realm should be replaced with Organization"},
```

Comment:
This should be a bit clearer, explaining the reason is that we use a nice user-facing name for it in user docs.